content/tutorials/manage-the-rippled-server/troubleshooting/diagnosing-problems.md

```diff
@@ -112,6 +115,43 @@ It is normal for a `rippled` the server to print many warning-level (`WRN`) mess
 For a more thorough explanation of various types of log messages, see [Understanding Log Messages](understanding-log-messages.html).
 
 
+## Info Collection Script
+
+If you have problems diagnosing the problem, or you are unable to resolve the problem with any of the common fixes, you may want to ask for help in a support forum or the [GitHub issues](https://github.com/ripple/rippled/issues). When asking for help, you can use an info collection script to gather information about your system to help others diagnose the issue.
+
+The official package installation (for [Ubuntu/Debian](install-rippled-on-ubuntu.html) or [CentOS/RedHat](install-rippled-on-centos-rhel-with-yum.html)) installs such a script by default, to `/opt/ripple/bin/getRippledInfo`. [New in: rippled 1.5.0][] If you compiled `rippled` yourself, you can find the same script [in the `rippled` source code repository](https://github.com/ripple/rippled/blob/develop/bin/getRippledInfo).
+
+To use the script:
+
+1. Run the script while `rippled` is running.
+
+        $ /opt/ripple/bin/getRippledInfo
+
+        ####################################################
+          rippled info has been gathered. Please copy the
+          contents of /tmp/ripple_info.Xo8Xr/rippled_info.md
+          to a github gist at https://gist.github.com/
+
+          PLEASE REVIEW THIS FILE FOR ANY SENSITIVE DATA
+          BEFORE POSTING! We have tried our best to omit
+          any sensitive information from this file, but you
+          should verify before posting.
+        ####################################################
+
+    The script collects the output of numerous commands and writes them to a temporary file. The filename is randomized with a string of letters and numbers (case-sensitive), for example: `/tmp/ripple_info.Xo8Xr/rippled_info.md`
+
+
+2. Look over the output file for sensitive information.
+
+    Although the script attempts to scrub sensitive information from the information it collects, it's important to look it over to be sure before posting the output publicly. For example, the script outputs detailed information about your server hardware, and you may want to remove some sections for privacy reasons. Use a text editor to read the output file and to remove anything you don't want to post.
+
+        nano /tmp/ripple_info.Xo8Xr/rippled_info.md
+
+3. Upload the output file where others can see it.
+
+    For example, you can upload a file directly to [GitHub Gist](https://gist.github.com/), [Pastebin](https://pastebin.com/), or a similar service. If you are running `rippled` on a remote server, you may find it easier to first transfer the file to a machine with a web browser, using `scp` or a similar tool.
```

Comment:
Delete "For example" and change "a" to "the"

You can upload the file directly to [GitHub Gist](https://gist.github.com/), [Pastebin](https://pastebin.com/), or a similar service.

Reply:
It might be good to also state what type of information is in the output file. ATM, it simply says "system" information. Is that services that are part of the rippled installation? The servers and db that rippled is connected to?

Reply:
It's honestly a lot of information. And I would expect the exact scope may change without warning in future releases. Seems like some of it depends on whether the script can collect it from your installation (in other words, the script just omits stuff it can't query) as well.

I was thinking of attaching a sample output I'm not really sure I have a good sample to work with. In any case, people should be looking over the output before they upload it, so they'll see what it actually collected then.